service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -75,4 +84,34 @@ void shouldSendEmail() throws IOException, NotificationClientException {
         verify(notificationClient, times(1)).sendEmail(
             eq(HMCTS_COURT_SUBMISSION_TEMPLATE), eq(COURT_EMAIL_ADDRESS), eq(expectedParameters), eq("12345"));
     }
+
+    @Test
+    void shouldSendEmailToCafcass() throws IOException, NotificationClientException {
+        CallbackRequest request = callbackRequest();
+        final Map<String, String> expectedParameters = ImmutableMap.<String, String>builder()
+            .put("cafcass", CAFCASS_NAME)
+            .put("localAuthority", "Example Local Authority")
+            .put("dataPresent", "Yes")
+            .put("fullStop", "No")
+            .put("orders0", "^Emergency protection order")
+            .put("orders1", "")
+            .put("orders2", "")
+            .put("orders3", "")
+            .put("orders4", "")
+            .put("directionsAndInterim", "^Information on the whereabouts of the child")
+            .put("reference", "12345")
+            .put("caseUrl", "null/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345")
+            .build();
+
+        given(cafcassLookupConfiguration.getCafcass(LOCAL_AUTHORITY_CODE))
+            .willReturn(new CafcassLookupConfiguration.Cafcass(CAFCASS_NAME, CAFCASS_EMAIL_ADDRESS));
```

Comment:
Please static import `CafcassLookupConfiguration.Cafcass` so that we do not see `CafcassLookupConfiguration` part.